nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,71 @@
+---
+title: SQL Server Transport usage guidance
+summary: Usage guidance for SQL Server transport
+tags:
+- SQL Server
+---
+
+The SQL Server transport allows endpoints to exchange messages using the Microsoft SQL Server Database Engine. It implements queues on top of database tables and send / receive operations as `INSERT` and `DELETE` operations.
+
+Please note that the SQL Server transport does not use or depend on SQL Server Service Broker.
+
+## How it works
+
+SQL Server provides a central place to store queues and messages but the message queuing implementation resides entirely within the endpoint process. SQL Server transport is best thought of as a brokered transport like RabbitMQ rather than store-and-forward transport such as MSMQ.
+
+### Advantages of SQL Server Transport
+
+ * No additional licensing and training costs, as majority of Microsoft stack organizations already have SQL Server installed and have knowledge required to run it
+ * Great general-purpose tooling (SSMS)
+ * Free to start (Express edition)
+ * Scaling out of processing is simple because of support for competing consumers at the queue level
+
+## Deployment considerations
+
+### Introduction
+Any NServiceBus endpoint operates and manages different types of data, those are:
+ * Business data - data managed by NServiceBus user, usually via code executed from inside message handlers
+ * Persistence data - data managed by infrastructure including: saga data, timeout manager state and message driven pub/sub information
+ * Transport data - queues and messages managed by the transport
+
+SQL Server Transport manages transport data and puts no constraints on the type and configuration of storage technology used for persistence and business data. It can work with any of available persisters i.e. NHibernate and RavenDB, as well as any storage mechanisms used from inside message handlers.
```

Comment:
> It can work with any of available persisters i.e. NHibernate and RavenDB, as well as any storage mechanisms used from inside message handlers.

Separate the persistence mechanism from the transport?